packages/ui/src/views/repo/repo-sidebar/index.tsx

```diff
@@ -1,11 +1,14 @@
-import { BranchListProps, BranchSelector, Button, ButtonGroup, Icon, SearchFiles } from '@/components'
+import { Button, ButtonGroup, Icon, SearchFiles } from '@/components'
 import { SandboxLayout } from '@/views'
+import { BranchSelectorListProps } from '@views/repo'
+import { BranchSelector } from '@views/repo/components'
 
 interface RepoSidebarProps {
   hasHeader?: boolean
   hasSubHeader?: boolean
   selectedBranch: string
-  branchList: BranchListProps[] | undefined
+  branchList: BranchSelectorListProps[]
+  tagList: BranchSelectorListProps[]
```

Comment:
Would a better name for `BranchSelectorListProps` be `BranchSelectorListItem`? The `Props` suffix should only be used if the type for the props of a component and `BranchSelectorListProps` doesn't appear to be